dev/reference/performance/statistics.md

```diff
@@ -15,24 +15,81 @@ Based on the statistics, the TiDB optimizer chooses the most efficient query exe
+>
+> For quicker analysis, you can set `tidb_enable_fast_analyze` to `1` to turn on Quick Analysis. The default value for this parameter is `0`.
+>
+> When Quick Analysis is turned on, TiDB randomly samples approximately 10,000 rows of data to build statistics. Therefore, in the case of uneven data distribution or relatively small amount of data, the accuracy of statistical information will be relatively poor. It may lead to poor execution plans, such as choosing the wrong index. If the execution time of the normal `ANALYZE` statement is acceptable, it is recommended to turn off Quick Analysis.
+
+You can perform full collection using the following syntaxes.
+
+Collecting statistics of all the tables in `TableNameList`:
+
+{{< copyable "sql" >}}
 
 ```sql
 ANALYZE TABLE TableNameList [WITH NUM BUCKETS]
-> The statement collects statistics of all the tables in `TableNameList`. 
-> `WITH NUM BUCKETS` specifies the maximum number of buckets in the generated histogram.
+```
+
+`WITH NUM BUCKETS` specifies the maximum number of buckets in the generated histogram.
+
+Collecting statistics of the index columns on all `IndexNameList`s in `TableName`:
 
+{{< copyable "sql" >}}
+
+```sql
 ANALYZE TABLE TableName INDEX [IndexNameList] [WITH NUM BUCKETS]
-> The statement collects statistics of the index columns on all `IndexNameList`s in `TableName`.
-> The statement collects statistics of all index columns when `IndexNameList` is empty.
+```
+
+The statement collects statistics of all index columns when `IndexNameList` is empty.
 
+Collecting statistics of partition in all `PartitionNameList`s in `TableName`:
+
+{{< copyable "sql" >}}
+
+```sql
 ANALYZE TABLE TableName PARTITION PartitionNameList [WITH NUM BUCKETS]
-> The statement collects partition statistics of all `PartitionNameList`s in `TableName`.
+```
 
+Collecting statistics of index columns for the partitions in all `PartitionNameList`s in `TableName`:
+
+{{< copyable "sql" >}}
+
+```sql
 ANALYZE TABLE TableName PARTITION PartitionNameList [IndexNameList] [WITH NUM BUCKETS]
-> The statement collects index column statistics of the partitions in all `PartitionNameList`s in `TableName`.
+```
+
+#### Incremental collection
+
+To improve the speed of analysis after full collection, incremental collection could be used to analyze the newly added sections in monotonically non-decreasing columns such as time columns.
+
+> **Note:**
+>
+> 1. Currently, incremental collection is only provided for index.
+> 2. When using incremental collection, you must ensure that there is only `INSERT` operations on the table, and that the newly inserted value on the index column is monotonically non-decreasing. Otherwise, the statistical information may be inaccurate, affecting the TiDB optimizer to select an appropriate execution plan.
```

Comment:
```suggestion
> - When using the incremental collection, you must ensure that only `INSERT` operations exist on the table, and that the newly inserted value on the index column is monotonically non-decreasing. Otherwise, the statistical information might be inaccurate, affecting the TiDB optimizer to select an appropriate execution plan.
```

Reply:
For "monotonically non-decreasing", please double check with @lamxTyler whether it's a typo of "单调递减". Because "单调不减" sounds a little bit weird.

Reply:
I've checked with @lamxTyler . "单调不减" is right.

Reply:
That's good.